docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md

```diff
@@ -488,18 +488,18 @@ In this section, we will:
   $(element).on('dropon', function(ev, drop, drag) {...});
   ```
 
-  Notice that `drag` is the 3nd argument to the event.  You can listen to
-  `drop` events in [can-stache] and pass the `drag` argument to a function like:
+  Notice that `drag` is the 2nd argument to the event.  You can listen to
```

Comment:
I think the arguments are `ev, drop, drag` so `drag` is the third argument. Am I reading the code incorrectly?